keras_core/ops/math.py

```diff
@@ -696,10 +698,10 @@ class Rsqrt(Operation):
 
     Example:
 
-    >>> x = keras_core.ops.convert_to_tensor([2., 3., -2.])
-    >>> rsqrt(x)
-    """
-
+   >>> data = keras_core.ops.convert_to_tensor([1.0,10.0,100.0])
+   >>> keras_core.ops.rsqrt(data)
+   Array([1.        , 0.31622776, 0.1       ], dtype=float32)
```

Comment:
Use `array` (lowercase) and trim whitespace

Reply:
Done